src/methods/mask.jl

```diff
@@ -1,4 +1,9 @@
 
+const INVERSE_KEYWORD = """
+- `inverse`: invert the mask, so that areas no missing in `with` are
+    masked, and areas missing in `with` are masked.
```

Comment:
Do the other packages also use `inverse`?  I feel like this should be `invert`...

Reply:
Me too! But raster/terra do.

Turns out its `invert` in rasterio so I'll switch it

Reply:
Maybe mention
```suggestion
const INVERSE_KEYWORD = """
- `inverse`: invert the mask, so that areas no missing in `with` are
    masked, and areas missing in `with` are masked.  Equivalent to 
    `inverse` in R's `raster` and `terra` packages.